roaring/roaring.go

```diff
@@ -101,6 +102,14 @@ func (b *Bitmap) Add(a ...uint64) (changed bool, err error) {
 
 	return changed, nil
 }
+func (b *Bitmap) String() string {
```

Comment:
1. maybe move this function before `Slice()` in the code. it's weird to have it dropped in the middle of the two `Add, add` functions.
2. it seems odd to include `\n` at the end of a `String()` function
3. https://github.com/pilosa/pilosa/blob/master/cache.go#L396 contains a similar `String()` function that doesn't have an extra space on the final iteration.